server/integration_tests/test_data/e2e_triple/whatistheprevalenceofheartdiseaseincalifornia/chart_config.json

```diff
@@ -416,494 +416,14 @@
               }
             ],
             "title": "Diabetes in Counties of California"
-          },
-          {
```

Comment:
curious why we lost this?

Reply:
This is due to the update of stat var descriptions. and these variables get even lower score. Since these are mostly unrelated variables, I would say this is a win?